gnomad/utils/filtering.py

```diff
@@ -529,3 +529,57 @@ def split_vds_by_strata(
     )
 
     return [hl.vds.filter_samples(vds, list(s)) for strata, s in s_by_strata.items()]
+
+
+def remove_items_from_freq(
+    ht: hl.Table, items_to_remove: Union[Dict[str, List[Any]], List[Any]]
+) -> hl.Table:
+    """
+    Script to remove items from the freq array and freq_meta array in the Table.
+
+    :param ht: Input Table with freq and freq_meta arrays.
+    :param items_to_remove: Dictionary or list of items to remove from the freq and freq_meta arrays.
+    :return: Table with specified items removed from the freq array and freq_meta array.
+    """
+
+    def _remove_key_value_pair_from_freq(
+        ht: hl.Table,
+        key: str,
+        value: str,
+    ) -> hl.Table:
+        """
+        Remove key-value pair from freq and freq_meta arrays.
+
+        :param key: Key to remove from freq_meta array.
+        :param value: Value to remove from freq_meta array.
+        :param ht: Input Table with freq and freq_meta arrays.
+        :return: Table with specified key-value pair removed from freq and freq_meta arrays.
+        """
+        freq = hl.map(lambda x: x[0].annotate(meta=x[1]), hl.zip(ht.freq, ht.freq_meta))
```

Comment:
Since this is a temp annotation add `_`

```suggestion
        freq = hl.map(lambda x: x[0].annotate(_meta=x[1]), hl.zip(ht.freq, ht.freq_meta))
```